oauth2/src/main/java/com/okta/spring/boot/oauth/env/OktaOAuth2PropertiesMappingEnvironmentPostProcessor.java

```diff
@@ -257,13 +288,13 @@ public Object getProperty(@NotNull String key) {
                 // issuer could be null (only resolve properties after checking if the key is `OKTA_OAUTH_ISSUER_WITH_PATH`)
                 return Optional.ofNullable(environment.getProperty(OKTA_OAUTH_ISSUER))
                     .map(issuer -> {
-                        // Check if URL is a Okta Org Authorization Server
-                        if (!issuer.contains("/oauth2")) {
-                            // for the Okta Org Authorization Server, /oauth2 needs to be appended
-                            issuer += "/oauth2";
+                        // Check if URL is an Auth0 org or is an Okta Org Authorization Server
+                        if (issuer.contains("auth0.com") || issuer.contains("/oauth2")) {
```

Comment:
`issuer` URL could contain both `auth0.com` and `/oauth2` 🤔

Reply:
should be ok since its `||` right